src/reducers/dataReducer.js

```diff
@@ -134,12 +133,10 @@ export function GRIDDLE_TOGGLE_COLUMN(state, action) {
 }
 
 export function GRIDDLE_UPDATE_STATE(state, action) {
-  const transformedData = transformData(action.newState.data, state.get('renderProperties').toJSON());
-  const data = transformedData.data;
-  const lookup = transformedData.lookup;
-  const newState = _.omit(action.newState, data);
```

Comment:
_.omit() accepts an array and then tries to unset the each value in the array from the object